.dockerignore

```diff
@@ -0,0 +1,15 @@
+node_modules
+*.aes
+*.decrypted
+codeship_deploy_key*
+codeship.env
+.dockerignore
+Dockerfile
+codeship*
+*.encrypted
+dist/
+lib/
```

Comment:
from what I understood we should not put `lib` into .dockerignore. @omgaz can you give us a bit of context on that please and how critical is it?

Reply:
the lib folder is used when we do a build. We only need to omit from the dockerfile when we use volumes in the services to mount a folder between steps e.g. build in one step and deploy in another.

I think it's valid to have here, but can't recall if this repo even uses `lib` as the build folder - maybe it's redundant.

Reply:
I see 👍  I'm glad I asked - will be applying to my future PRs 🙂